src/bokeh/models/formatters.py

```diff
@@ -741,10 +741,32 @@ def CONTEXTUAL_DATETIME_FORMATTER() -> DatetimeTickFormatter:
 #-----------------------------------------------------------------------------
 
 # This is to automate documentation of DatetimeTickFormatter formats and their defaults
-_dttf = DatetimeTickFormatter()
-_dttf_fields = ('microseconds', 'milliseconds', 'seconds', 'minsec', 'minutes', 'hourmin', 'hours', 'days', 'months', 'years')
-_dttf_defaults = _dttf.properties_with_values()
-_dttf_defaults_string = "\n\n        ".join(f"{name} = {_dttf_defaults[name]!r}" for name in _dttf_fields)
+_dttf = CONTEXTUAL_DATETIME_FORMATTER()
+
+_dttf_fields = ('microseconds', 'milliseconds', 'seconds', 'minsec', 'minutes', 
+                'hourmin', 'hours', 'days', 'months', 'years')
+
+def create_format_table() -> str:
+    # Header
+    table = "Scale         Format           1st Context    2nd Context\n"
+    table += "-" * 56 + "\n"  # Separator line
+    
+    # Get formatters for each context level
+    primary = _dttf
+    context1 = _dttf.context
+    context2 = _dttf.context.context if _dttf.context else None
+    
+    # Build table rows
+    for field in _dttf_fields:
+        scale = f"{field:<12}" 
+        p_fmt = f"{getattr(primary, field):<15}"
+        c1_fmt = f"{getattr(context1, field):<15}" if context1 else " "*15
+        c2_fmt = f"{getattr(context2, field):<15}" if context2 else " "*15
+        table += f"{scale} {p_fmt} {c1_fmt} {c2_fmt}\n"
+    
+    return table
+
+_dttf_defaults_string = create_format_table()
 
 DatetimeTickFormatter.__doc__ = format_docstring(DatetimeTickFormatter.__doc__, defaults=_dttf_defaults_string)
```

Comment:
It is easier to write this in one line.

```suggestion
DatetimeTickFormatter.__doc__ = format_docstring(DatetimeTickFormatter.__doc__, defaults=create_format_table())
```